src/components/ChartContextMenu.svelte

```diff
@@ -48,6 +49,34 @@
 
   const dateFormatter = timeFormat('%Y-%m-%d');
   const timeFormatter = timeFormat('%H:%M:%S');
+
+  const getTelemetryPath = () => {
+    if ($store.probe.type === 'histogram') {
+      return ['main', 'parent'].includes($store.productDimensions.process)
+        ? `payload.histograms.${$store.probe.name}`
+        : `payload.processes.${$store.productDimensions.process}.histograms.${$store.probe.name}`;
+    }
+    return undefined;
+  };
+
+  const table =
+    $store.productDimensions.channel === 'nightly'
+      ? 'main_nightly'
+      : 'main_1pct';
+
+  const getComparisonViewinSTMO = () =>
+    $store.product === 'firefox'
+      ? 'https://sql.telemetry.mozilla.org/queries/82247/source?' +
```

Comment:
I think this would be slightly easier to read if you constructed it with [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)